ui/src/app/views/workspace/component.spec.ts

```diff
@@ -21,8 +29,13 @@ import {
   simulateClick,
   updateAndTick
 } from 'testing/test-helpers';
+import {ServerConfigServiceStub} from '../../../testing/stubs/server-config-service-stub';
+import {ServerConfigService} from '../../services/server-config.service';
 
 class WorkspacePage {
```

Comment:
SHould there be a test which covers opening the modal?

Reply:
Since it's the modal is now getting opened via the workspace-nav-bar, I would think the test would go there.